test/nl/hannahsten/texifyidea/formatting/LatexFormattingTest.kt

```diff
@@ -185,6 +185,22 @@ fun Int?.ifPositiveAddTwo(): Int =
             \end{algorithm}
         """.trimIndent()
     }
+//
+//    fun testComments() {
+//        // Wanted to test line breaking, but not sure how to enable it in test
+//        val text = """
+//            % Calculated protections are able to develop anxious insurances when they prick notes and relate identities and rejects.
+//        """.trimIndent()
+//        val file = myFixture.configureByText(LatexFileType, text)
+//        CodeStyle.getLanguageSettings(file).RIGHT_MARGIN = 50
+//        CodeStyle.getLanguageSettings(file).WRAP_ON_TYPING = CommonCodeStyleSettings.WrapOnTyping.WRAP.intValue
+//        CodeStyle.getDefaultSettings().WRAP_WHEN_TYPING_REACHES_RIGHT_MARGIN = true
+//        writeCommand(project) { CodeStyleManager.getInstance(project).reformat(myFixture.file) }
+//        val expected = """
+//            % Calculated protections are able to develop anxious insurances when they prick notes and relate identities and rejects.
+//        """.trimIndent()
+//        myFixture.checkResult(expected)
+//    }
```

Comment:
hm.. looking at the code you'd expect this to work. I'd fix it or get rid of it before merging to master? now it's just comment.

Reply:
So far I kept commented out tests, because it sometimes happened that I found out later how to do a certain thing in tests and then I could easily find back all the commented out tests in which that applies, or sometimes I run into a commented out test for which I know how to fix it.
I could also create a wiki page with "tests that should work but don't", but then I need to remember that I put it there